deny.toml

```diff
@@ -53,4 +53,6 @@ ignore = [
 deny = [
     # https://github.com/vectordotdev/vector/issues/11257
     { name = "tokio-util", version = ">=0.6.9" },
+    # https://github.com/vectordotdev/vector/pull/12865
+    { name = "goauth", version = "=0.13.0" },
```

Comment:
I might have made this `>=` in case we skip to 0.14.0 and it has the same issue.